wrangler-service/src/main/java/io/cdap/wrangler/service/directive/WorkspaceHandler.java

```diff
@@ -253,6 +261,25 @@ public void execute(HttpServiceRequest request, HttpServiceResponder responder,
     });
   }
 
+  private List<Row> executeLocally(String namespace, List<String> directives,
+                                   WorkspaceDetail detail) throws DirectiveLoadException, DirectiveParseException {
+    // load the udd
+    composite.reload(namespace);
+    return executeDirectives(namespace, directives, new ArrayList<>(detail.getSample()));
+  }
+
+
+  private List<Row> executeRemotely(String namespace, List<String> directives,
+                                    WorkspaceDetail detail) throws Exception {
+    LOG.info("Executing remotely");
```

Comment:
Is this called everytime a new directive is added? should we reduce log level?

Reply:
Will remove this, was added for test